src/prefect/environments/storage/local.py

```diff
@@ -45,7 +44,8 @@ def __init__(self, directory: str = None, validate: bool = True) -> None:
             abs_directory = directory
 
         self.directory = abs_directory
-        super().__init__()
+        result_handler = LocalResultHandler(self.directory, validate=validate)
```

Comment:
This will write results to the `.prefect/flows` directory. Thoughts? I think it should have it's own separate directory for results

Reply:
It does `os.path.join(prefect.config.home_dir, "results")` but only if there is no directory specified

Reply:
Oh wait I see that it does `commonpath`, nvm